src/app/common/CodeBlock.vue

```diff
@@ -74,22 +87,21 @@ async function handleCodeBlockRenderEvent({ preElement, codeElement, language, c
 }
 </script>
 
-<style lang="scss">
-.code-block code {
-  // DO NOT REMOVE OR OVERRIDE THIS.
-  // Believe it or not, this avoids a critical performance issue with highlighting code and specifically highlighting lines using Prism’s line-highlight plugin.
-  // https://github.com/PrismJS/prism/issues/2062
-  display: block !important;
```

Comment:
**Question**: Have you tested and verified that this is not an issue anymore?

Reply:
I played with it in Chrome and whilst I've never seen the issue before, I didn't notice any issues when removing this.

I also noticed from reading linked GH issue in the comment (which is from 2019, but runs to 2022 / Chrome 77 to 88, so roughly between 5 and 2 years old) that it seems to have been fixed, possibly regressed and then fixed, its kinda hard to tell. In any case its clear it was a browser bug, and as I'm not noticing anything terrible rn, it seems fine to me.

Could you have a look yourself and see if you notice anything? I'd be fine adding this back in if so. lmk

btw, have you ever seen this bug/performance issue yourself? If so, could you explain how to reproduce?

Reply:
> btw, have you ever seen this bug/performance issue yourself? If so, could you explain how to reproduce?

Yes. When I built the code block component, I noticed this effect. The performance implication then was massive (echoing the descriptions in the issue).

I’m gonna do my own test (also making sure nothing else sets `display: block` on the `code` element).

To reproduce, all you would do then was run any query (or a series of queries) in the code block. The syntax highlighting became so slow that the search itself appeared to be extremely slow (which wasn’t the case). You’d definitely notice it if it still happens.

Reply:
@kleinfreund did you get a chance to look at the CodeBlock in Chrome? I've tried again since and not noticed anything.

Apart from that was there anything else blocking on this PR?

FYI I'm waiting on this PR before I do the module move and then i18n everything.

Reply:
P.S. FYI, I also just realized I kinda need this for the other work I'm doing now also.

I'm looking to reuse the `.../traffic` DataSource on all our stats pages, which means I can't use `EnvoyData` (the actual stats are no longer just `data` but `data.raw`). This was actually one of the reasons I did this PR - so that I could use CodeBlock everywhere and feed it things shaped in all different ways.

Reply:
I couldn’t reproduce this issue anymore, so I’m gonna resolve this.